File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/schema/RateRequest.java
package ru.iqchannels.sdk.schema;


import androidx.annotation.NonNull;

/**
 * Copyright iqstore.ru. All Rights Reserved.
 */
public class RateRequest {
    public long RatingId;
    @NonNull public RatingInput Rating;

    public RateRequest() {}

    public RateRequest(long ratingId, int value) {
        this.RatingId = ratingId;
        this.Rating = new RatingInput(value);
    }
}

File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/schema/RateRequest.kt
package ru.iqchannels.sdk.schema

class RateRequest {
	var RatingId: Long = 0
	var Rating: RatingInput? = null

	constructor()
	constructor(ratingId: Long, value: Int) {
		RatingId = ratingId
		Rating = RatingInput(value)
	}
}
